### ISSUE TITLE
Shutdown all mounts If no communication with WIS

### ISSUE THREAD

**Issue by EranOfek** [blumzi/LAST_issues#47]
Need to automatically shutdown the mounts and close the roof if no communication with WIS for more than 1 hour.

**Comment by EastEriq** [comment]
yes and no. If meteorological safety persists, observations can continue alone till morning. And users on site will want an override to the automatism.

In any event this is a task for a (local Unit or sitewise) supervisor, not for the unitCS code itself. Let that be clear.
